admin/src/views/AdminSignup.vue

```diff
@@ -147,7 +171,7 @@ export default {
           if (adminSignupResponse.data.createdUser) {
             this.$router.push({
               name: 'Login',
-              params: { successFeedback: true },
+              params: { successFeedback: 'true' },
```

Comment:
hmm, why was this changed to a string? line 117 references it as a boolean 🤔

Reply:
True, will change that